docs/text.mdx

```diff
@@ -3,9 +3,9 @@ name: Text
 ---
 
 import { Playground, PropsTable } from 'docz'
-import Text from 'components/text'
-import View from 'components/view'
-import Button from 'components/button'
+import Text from '@emcasa/ui-dom/components/text'
+import View from '@emcasa/ui-dom/components/view'
+import Button from '@emcasa/ui-dom/components/button'
```

Comment:
you can set `components` folder as the `main` folder in the `package.json` to make imports shorter such as `@emcasa/ui/text`